modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java

```diff
@@ -51,6 +53,34 @@ public AccountRole addAccountRole(
 		return new AccountRoleImpl(role);
 	}
 
+	@Override
+	public void addAccountRoleUser(
+			long accountEntryId, long roleId, long userId)
+		throws PortalException {
+
+		AccountEntry accountEntry = _accountEntryLocalService.getAccountEntry(
+			accountEntryId);
+
+		_userGroupRoleLocalService.addUserGroupRoles(
+			userId, accountEntry.getAccountEntryGroupId(), new long[] {roleId});
+	}
+
+	@Override
+	public List<AccountRole> getAccountRoles(long accountEntryId, long userId)
```

Comment:
i think it's more clear if this renames to `getUserAccountRoles`

Reply:
Haha, that's what I originally called it. I'll float it up to Brian to confirm the name.